apps/mgmt-ui/src/hooks/useCustomers.ts

```diff
@@ -3,11 +3,12 @@ import { CustomerExpandedSafe } from '@supaglue/core/types/customer';
 import { useSWRWithApplication } from './useSWRWithApplication';
 
 export function useCustomers() {
-  const { data, isLoading, error } = useSWRWithApplication<CustomerExpandedSafe[]>('/api/internal/customers');
+  const { data, isLoading, error, ...rest } = useSWRWithApplication<CustomerExpandedSafe[]>('/api/internal/customers');
```

Comment:
Seems like we should just remove `isLa
```suggestion
  const { data, isLoading, error, ...rest } = useSWRWithApplication<CustomerExpandedSafe[]>('/api/internal/customers');
```
```suggestion
  const { data, ...rest } = useSWRWithApplication<CustomerExpandedSafe[]>('/api/internal/customers');
```